src/pybind/mgr/dashboard/frontend/src/app/shared/services/api-interceptor.service.ts

```diff
@@ -59,9 +59,6 @@ export class ApiInterceptorService implements HttpInterceptor {
             case 403:
               this.router.navigate(['/403']);
               break;
-            case 404:
```

Comment:
Please adapt the unit tests. Test 'should redirect 404' fails because it is obsolete now.

Reply:
Looks like a test has sneaked in in the meanwhile!